build.sbt

```
@@ -224,3 +230,36 @@ lazy val scalafix = tlScalafixProject
     startYear := Some(2023),
     crossScalaVersions := Seq(Scala212)
   )
+
+lazy val googleCloudHttp4s = crossProject(JSPlatform, JVMPlatform)
+  .in(file("google-cloud-http4s"))
+  .settings(
+    name := "feral-google-cloud-http4s",
+    libraryDependencies ++= Seq(
+      "org.typelevel" %%% "cats-effect" % catsEffectVersion,
+      "org.scodec" %%% "scodec-bits" % "1.2.0",
+      "org.http4s" %%% "http4s-server" % http4sVersion,
+      "org.scalameta" %%% "munit-scalacheck" % munitVersion % Test,
+      "org.typelevel" %%% "munit-cats-effect-3" % munitCEVersion % Test
+    ),
+    mimaBinaryIssueFilters ++= Seq(
+      ProblemFilters.exclude[IncompatibleResultTypeProblem](
+        "feral.google-cloud.IOLambda.setupMemo")
+    ),
+    tlVersionIntroduced := List("2.13", "3").map(_ -> "0.3.1").toMap
+  )
+  .settings(commonSettings)
+  .jsSettings(
+    libraryDependencies ++= Seq(
+      "io.circe" %%% "circe-scalajs" % circeVersion,
+      "io.github.cquiroz" %%% "scala-java-time" % "2.5.0"
+    )
+  )
+  .jvmSettings(
+    Test / fork := true,
+    libraryDependencies ++= Seq(
+      "com.google.cloud.functions" % "functions-framework-api" % "1.1.0",
+      "co.fs2" %%% "fs2-io" % fs2Version,
+      "com.google.cloud.functions.invoker" % "java-function-invoker" % "1.3.1"
```

This dependency is "provided" by Google Cloud when we deploy the function. We only need to add it when we are testing the function locally.

```suggestion
      "com.google.cloud.functions.invoker" % "java-function-invoker" % "1.3.1" % Provided
```

---

Turns out, this was wrong: the `functions-framework-api` should be `Provided` according to the docs, and the `invoker` should be moved to the example project because whether or not its needed at runtime depends _where_ you are running it.